pkg/api/handlers/generic/containers_top.go

```diff
@@ -21,40 +22,40 @@ func TopContainer(w http.ResponseWriter, r *http.Request) {
 		PsArgs: "-ef",
 	}
 	if err := decoder.Decode(&query, r.URL.Query()); err != nil {
-		Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest,
+		handlers.Error(w, http.StatusText(http.StatusBadRequest), http.StatusBadRequest,
 			errors.Wrapf(err, "Failed to parse parameters for %s", r.URL.String()))
 		return
 	}
 
 	name := mux.Vars(r)["name"]
 	ctnr, err := runtime.LookupContainer(name)
 	if err != nil {
-		ContainerNotFound(w, name, err)
+		handlers.ContainerNotFound(w, name, err)
 		return
 	}
 
 	state, err := ctnr.State()
 	if err != nil {
-		InternalServerError(w, err)
+		handlers.InternalServerError(w, err)
 		return
 	}
 	if state != define.ContainerStateRunning {
-		ContainerNotRunning(w, name, errors.Errorf("Container %s must be running to perform top operation", name))
+		handlers.ContainerNotRunning(w, name, errors.Errorf("Container %s must be running to perform top operation", name))
```

Comment:
No capitalized errors.

Reply:
to be consistent with the other api, we must use caps.